content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service:_Deployment_Guide/Prerequisites_and_Preparations.adoc

```diff
@@ -0,0 +1,27 @@
+== Prerequisites and Preparations
+
+The audience of this document is an experienced systems administrator with a good understanding of Microsoft Hyper-V virtualization management. In addition, it is helpful to be familiar with the terminology, software and tools specific to YubiHSM 2. As a primer for these, refer to link:Terminology.adoc[the Terminology chapter in this guide].
+
+In order to follow the steps provided in this guide, be sure to meet the following prerequisites:
+
+* Microsoft Windows Server 2016 or higher. The operating system should be installed in a secure computer network. The system administrator must also have elevated system privileges.
+
+* YubiHSM 2 software and tools for Windows downloaded from the link:https://developers.yubico.com/YubiHSM2/Releases/[Yubico YubiHSM 2 Release page] and available on the system to be used.
+
+* Two (2) YubiHSM 2 devices, one for deployment and one for backup in hardware.
+
+* Your organization's policies may require key custodians to be available for the YubiHSM 2 deployment. For more information about key custodians and the associated ‘M of N’ key shares, see link:../YubiHSM_2_Windows_Deployment_Guide/Prerequisites_and_Preparations.adoc#key-splitting-and-key-custodians["Key Splitting and Key Custodians" in the YubiHSM 2 Windows Deployment Guide].
```

Comment:
This link is broken, the directory name has changed.

Reply:
I was trying to link to a heading in the file, but evidently that does not work. I have corrected the link to target just the file.

Reply:
the issue with the link is the directory name, it moved in another commit part of this PR, the link should be something like `link:../YubiHSM_2_Windows_Deployment_Guide\--Configure_YubiHSM_2_Key_Storage_Provider_for_Microsoft_Windows_Server/Prerequisites_and_Preparations.adoc`